server/api/routes/company.js

```diff
@@ -1,23 +1,34 @@
 const express = require('express')
 const router = express.Router()
-const { getCompany } = require('../../data_access_layer/company')
+const { getUserByCompanyCode } = require('../../data_access_layer/company')
 
 router.get('/:code', async (req, res) => {
   try {
     const { code } = req.params
-    const company = await getCompany(code)
-    if (company == null) {
+    const user = await getUserByCompanyCode(code)
+    if (user == null) {
       throw {
         statusCode: 404,
-        message: `Company with ID: ${code} not found.`
+        message: `No user exists with the company code '${code}'.`

```

Comment:
can you update this to be the same as the way 409 is returned, like instead of `throw` it just `return`